spec/models/user_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { create(:user) }
+
+  it { is_expected.to validate_presence_of(:name) }
+  it { is_expected.to validate_length_of(:name).is_at_least(1) }
+
+  it { is_expected.to validate_presence_of(:email) }
+  it { is_expected.to validate_uniqueness_of(:email) }
+  it { is_expected.to validate_length_of(:email).is_at_least(3) }
+  it { is_expected.to allow_value("user@bloccit.com").for(:email) }
+
+  it { is_expected.to validate_presence_of(:password) }
+  it { is_expected.to validate_length_of(:password).is_at_least(6) }
+
+
+  describe "attributes" do
+    it "should have name, email, and password attributes" do
```

Comment:
So is this more of a syntax point? Should this read `it "has name, email, and password attributes"`?